scylla-cdc-replicator/src/replication_tests.rs

```diff
@@ -0,0 +1,129 @@
+use futures_util::StreamExt;
+use itertools::Itertools;
+use scylla::frame::response::result::{CqlValue, Row};
+use scylla::transport::topology::CqlType;
+use scylla::{Session, SessionBuilder};
+
+pub type TestColumn = (String, CqlType);
+
+pub struct TestTableSchema {
+    name: String,
+    partition_key: Vec<TestColumn>,
+    clustering_key: Vec<TestColumn>,
+    other_columns: Vec<TestColumn>,
+}
+
+// [TODO]: We might want to provide queries for common operations like insert or update with one value.
+type TestOperation = (String, Vec<CqlValue>);
+
+fn get_type_name(col_type: &CqlType) -> &str {
```

Comment:
This function might become large if fully implemented (there are many native types and we would have to handle all of them). Could we switch from `CqlType` in `TestColumn` to just `String`, for simplicity?